### DISCUSSION TITLE
M1 Chip compatibility

### DISCUSSION THREAD

**Discussion by cRiTza** [apohl79/audiogridder#436] [Q&A]
Hello i have a mac mini m1, can i contribute in any way to test or develop your software!?

best regards Christer

**Comment by apohl79** [comment]
Hey, for starters you can see if the arm64 versions work for you and are stable. :-)

Regarding development contributions, i think i will have to make the build system a bit more friendly to others and put together a guide on how to set it up. If you want to get involved, you can have a look at the vst2 discussion thread.

**Comment by cRiTza** [comment]
Hey ok cool, i could test it out.  Thnx for the cool product!

Best regards Christer

**Comment by cRiTza** [comment]
Hello i downloaded the 1.2.0 beta 3 and it wouldnt connect by itself. One
windows 10 Pro, OS Version 19041.1052
And OSX 11.4 M1 Chip

Version 1.1.1 Works but i have had 2 crashes/hangups since i started the
program.
First hangup seemed to be on the serverplugin when i registered the plugin
(u-he Hive 2) on the m1 machine, and i could not force quit the program
since it didnt show up in Force quit menu.
Second hangup was when i loaded a second instance of Hive 2 on another
channel in the same session, i had to log out and into OSX and then the
settings / sounds i had got reset.

Could i send you a report of some sort? and if so how do i go about doing
that?

Best regards CHrister

Den lör 13 mars 2021 kl 13:54 skrev Christer Jonsson ***@***.***>:

**Reply by apohl79** [reply]
please follow https://audiogridder.com/bug-reports/ 

thx

**Comment by cRiTza** [comment]
Hello how is your development going?
I havent seen any updates lately

Best regards Christer
